okapi-core/src/main/java/org/folio/okapi/MainCluster.java

```diff
@@ -10,12 +9,14 @@ private MainCluster() {
   }
 
   public static void main(String[] args) {
+    System.setProperty("vertx.logger-delegate-factory-class-name",
+     "io.vertx.core.logging.Log4jLogDelegateFactory");
     Logger logger = OkapiLogger.get();
     MainDeploy d = new MainDeploy();
     d.init(args, res -> {
       if (res.failed()) {
-        logger.error(res.cause().getMessage(), res.cause());
-        exit(1);
+        logger.error(res.cause().getMessage());
```

Comment:
Removing `res.cause()` swallows the stacktrace. Why?

Reply:
No stacktrace.. It says, for example, io.vertx.core.impl.NoStackTraceThrowable: Unknown command: xx , so it seems useless to me